CodeStarterCamp_Week3/main.swift

```diff
@@ -10,6 +10,14 @@ import Foundation
 
 var misterLee = Person(name: "misterLee", money: 3000)
 var missKim = Person(name: "missKim", money: 5000)
-var yagombucks = CoffeeShop(menu: [Coffee.americano: 3500, Coffee.decafAmericano: 4000, Coffee.vanillaLatte: 3800, Coffee.hazelnutLatte: 3800, Coffee.caramelmacchiato: 3900], barista: misterLee)
+let yagombucksMenu: [Coffee: Int] = [
+    .americano: 3500,
+    .decafAmericano: 4000,
+    .vanillaLatte: 3800,
+    .caramelmacchiato: 3900
+]
+var yagombucks = CoffeeShop(name: "yagombucks", menu: yagombucksMenu, barista: misterLee)
 
-missKim.order(Coffee.americano, at: yagombucks)
+missKim.order(Coffee.americano, of: yagombucks, by: missKim.name)
```

Comment:
요 `Coffee.americano`역시 우리가 함수의 선언에서 매개변수의 타입을 지정해줬기 때문에, 타입을 생략할 수 있습니다 😁
```suggestion
missKim.order(.americano, of: yagombucks, by: missKim.name)
```